include/swift/AST/Module.h

```diff
@@ -857,7 +870,9 @@ class SourceFile final : public FileUnit {
   /// This is the list of modules that are imported by this module.
   ///
   /// This is filled in by the Name Binding phase.
-  ArrayRef<std::pair<ModuleDecl::ImportedModule, ImportOptions>> Imports;
+  ArrayRef<std::pair<ModuleDecl::ImportedModule,
+                     std::pair<ImportOptions, StringRef>>>
```

Comment:
Can you make a proper type or at least a typedef for this? Or put the StringRef into ImportOptions?